docs/src/scss/_buttons.scss

```diff
@@ -48,11 +48,14 @@ fieldset:disabled a.doc-button {
   color: $white;
   background-color: $primary-button-color;
   border-color: $primary-button-color;
+  box-shadow: 0px 11px 33px rgba(255, 220, 179, 0.62);
 
   &:hover {
     color: $white;
     background-color: $primary-button-hover-color;
     border-color: $primary-button-hover-color;
+    animation: pulse 1s;
+    box-shadow: 0 0 0 1.25em rgba(255,255,255,0),
```

Comment:
Can we be consistent in terms of units ? On line 51, it is given as px. Here it is em. It will be better if units are uniform.

Reply:
Changed all box-shadow units to `px`